ds3_java_cli/src/main/java/com/spectralogic/ds3cli/exceptions/CommandExceptionFactory.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.exceptions;
+
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CommandExceptionFactory {
```

Comment:
We should update the name of this to better reflect what it is actually responsible for.  Specifically, this is the ExceptionHandlerMapper.  It's not a Factory.

Also, some javadoc to explain what this class is doing/responsible for should be added.